common/util.go

```diff
@@ -431,12 +432,22 @@ func CreateHistoryStartWorkflowRequest(
 	domainID string,
 	startRequest *types.StartWorkflowExecutionRequest,
 	now time.Time,
-) *types.HistoryStartWorkflowExecutionRequest {
+) (resp *types.HistoryStartWorkflowExecutionRequest, retError error) {
```

Comment:
do we need named return parameter in this case, looks like `resp` is never used and a new `histRequest` is created and returned